### ISSUE TITLE
[Question] mismatch between bbox and image in RefCOCO

### ISSUE THREAD

**Issue by WeitaiKang** [TRI-ML/vlm-evaluation#16]
Hi authors,

Thanks for your great job!

However, for the evaluation in Visual Grounding (Refcoco/+/g), I find that the coordinate of your normalized bbox mismatch with the image processed by LLaVA1.5.

Specifically, your [code](https://github.com/TRI-ML/vlm-evaluation/blob/d9f58967de7f62371ee068dbfa07878cdaef5607/vlm_eval/tasks/harnesses/refcoco.py#L92) for bbox normalize the bbox based on the original image size. Instead, the image will [go through](https://github.com/TRI-ML/vlm-evaluation/blob/d9f58967de7f62371ee068dbfa07878cdaef5607/vlm_eval/models/llava.py#L54) LetterBoxPad and  resize to 336px. Therefore, the normalized bbox's coordinates don't match the pixels' coordinates of input image in LLaVA.

Isn't it a problem? Is it the same way of how LLaVA generate their training data?

**Comment by WeitaiKang** [comment]
According to [issue in LLaVA codebase](https://github.com/haotian-liu/LLaVA/issues/606#issuecomment-1774076879), the bbox in their training data has considered the LetterBoxPad and 336px. Therefore, I think your preparation in ground truth bbox might not be correct. How do you think?
